zendesk/credential.go

```diff
@@ -53,3 +59,35 @@ func (c APITokenCredential) Email() string {
 func (c APITokenCredential) Secret() string {
 	return c.apiToken
 }
+
+// Type is accessor which returns the type
+func (c APITokenCredential) Type() string {
```

Comment:
we need to change this to "Basic" as well

Reply:
Made the change 👍